README.Windows.md

```diff
@@ -0,0 +1,41 @@
+﻿## Windows specific info
+
+This document details differences in pre-requisites and building instructions under Windows.
```

Comment:
Can this in be added in the wiki rather than here? https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Dependencies

Reply:
Sure; I was just trying to be parallel to README.md. Keep in both, like for Linux?

Reply:
So the readme is a bit out of date at the moment, I've done work to try and make sure the wiki is up to date. I've utilized several pages in the wiki to make it a bit simpler. I was planning on making the front page readme a bit simpler and just using it to explain VW's mission and why use it, then link into the wiki for instructions